src/components/forecast/ForecastDetail.js

```diff
@@ -0,0 +1,110 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import {
+  BsFillCloudMoonFill,
+  BsFillCloudSunFill,
+  BsFillSunFill,
+  BsMoonFill,
+  BsCloudSnowFill,
+  BsCloudRainHeavyFill,
+} from 'react-icons/bs';
+import classes from './ForecastDetail.module.css';
+
+const apiKey = process.env.REACT_APP_KEY;
```

Comment:
<img width="331" alt="Screenshot 2023-01-06 at 04 47 49" src="https://user-images.githubusercontent.com/69011963/210901475-d7c0a871-b23e-4a04-988a-6533f25415a5.png">

[OPTIONAL] The search bar is functioning optimally, however, its current placement detracts from the overall design. I would suggest repositioning it accordingly.